### ISSUE TITLE
Missing property `BuiltModuleSubdirectory` in template for build.yml

### ISSUE THREAD

**Issue by johlju** [gaelcolas/Sampler#368]
This property should be removed as it does not do anything (no code support it):

https://github.com/gaelcolas/Sampler/blob/d19ef072498ac9df81a3839b0b7a502a066f2235/Sampler/Templates/Build/build.yaml.template#L11

Instead we should add the property `BuiltModuleSubdirectory`:

```
BuiltModuleSubdirectory: builtModule
```

**Comment by gaelcolas** [comment]
OutputDirectory is used, but I agree that BuiltModuleSubdirectory: should be there too.
Actually, it should be the default IMO.
Will send a PR for that.

**Comment by johlju** [comment]
OutputDirectory is used as a variable in the task, but if set in build.yaml it is never fetch by any code, at least not what I could find.

**Comment by gaelcolas** [comment]
Correct, it's only the `build.ps1` parameter. https://github.com/gaelcolas/Sampler/blob/main/build.ps1#L80
